contribute/en/Style-and-Tone.md

```diff
@@ -83,17 +83,19 @@ Refer to UI labels in inverted commas (eg 'click on the “mute” button')
 
 Jamulus is “Free and Open Source (FOSS)” (not “free software” or “open source”)
 
-“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together” (not “Mute a person” because one person might be using multiple channels).
+“Channel” The audio signal as part of a mix. “Mute a channel”, “Maximum number of channels”, “Group channels together”.
 
-“Fader” The UI that controls a channel. “Each fader has a mute button”, “The person’s fader” ,“Group faders together” (not “The person’s channel” or “Mute a Fader”, not “Slider” or “Volume control”)
+"Fader" - the UI control that adjusts the channel volume heard in the mix. Not "volume slider", "volume control" or other term (adding the context of the mixer, if needed).
 
-“Person” A human connected to a server (may be on multiple channels). We might say "People on the server", or "People who have muted themselves", rather than _Musicians_ or _Channels_.
+“Person” When "channel" or "fader" isn't a good term (according to the above definitions), we refer to "people" connected to a server rather than "musicians" since (in English) the latter term does not imply singers.
```

Comment:
I've settled on _musician_ but _person_ is ok. The reality is that it might be a bot, and so _connection_ is accurate, but being understood is more important in user docs than being accurate. This idea that singers aren't musicians is nothing I've heard of.

Reply:
> This idea that singers aren't musicians is nothing I've heard of

Me neither, but I'm not a native speaker.

Reply:
> I've settled on _musician_ but _person_ is ok. The reality is that it might be a bot, and so _connection_ is accurate, but being understood is more important in user docs than being accurate. This idea that singers aren't musicians is nothing I've heard of.

That would be news to [The Guild](https://www.musiciansandsingers.org.uk/) :-) But in our case it's to help in the translation of other languages, and also because you can have listeners and others in some contexts perhaps.

And yes, being understood is more important than accuracy here. The docs are the spec, the style guide is there to establish consistency and clarity where possible.

Reply:
> This is "ok" but quite broad. "Person" should always refer to the actual human being. A Person may have multiple channels etc.

Yes. This is why it says _"When "channel" or "fader" isn't a good term"_

Reply:
> The reality is that it might be a bot, and so _connection_ is accurate, but being understood is more important in user docs than being accurate. 

No, "Person" refers to a real human.  A bot might also have a connection but that's not what a Person refers to.  There can be more than one person on a single connection, too, so we have to be clear that "Person" is _not_ the same thing as "connection".